src/main/java/org/resitrack/controller/PersonController.java

```diff
@@ -105,6 +241,13 @@ private String generatePersonId() {
         return "P" + count.getAndIncrement();
     }
 
+    /**
+     * This function is used to add new person
+     *
+     * @param person
```

Comment:
please explain where is this param come from, what is value of it

Reply:
Đã update